stac_check/lint.py

```diff
@@ -18,15 +17,16 @@ class Linter:
     config_file: Optional[str] = None
     assets: bool = False
     links: bool = False
-    recursive: bool = False
+    recursive: int = 0
 
     def __post_init__(self):
         self.data = self.load_data(self.item)
         self.message = self.validate_file(self.item)
         self.config = self.parse_config(self.config_file)
         self.asset_type = self.message["asset_type"] if "asset_type" in self.message else ""
         self.version = self.message["version"] if "version" in self.message else ""
-        self.validator_version = "2.3.0"
+        self.validator_version = "3.0.0"
```

Comment:
Can we get this from `stac-validator`, e.g. via `stac_validator.__version__` or this more refined version:

```python
import pkg_resources
self.validator_version = pkg_resources.require("stac_validator")[0].version
```

Reply:
great tip thanks!